src/Shared/UnitTests/TestEnvironment.cs

```diff
@@ -583,7 +583,14 @@ public override void Revert()
         {
             if (_processId > -1)
             {
-                Process.GetProcessById(_processId).KillTree(1000);
+                try
+                {
+                    Process.GetProcessById(_processId).KillTree(1000);
+                }
+                catch
+                {
+                    // ignore if process is already dead
```

Comment:
Wondering about this...it'd be a bit messy to make it return a bool for "succeeded in really reverting something", and that wouldn't really help anyway because you don't explicitly call this in tests...but I think of these TransientTest___s as asserting first that it exists and second that they can make it not exist, and this breaks that assumption. Not sure what the right answer is.

Reply:
One of the goal of TransientTest__s is, IMO, to get into consistent state so previous tests side effects will not effect next test runs. So if process has exited himself, was killed by another logic, etc... we still get into consistent state => there is no left over process.
If test want to assert that process is still running at some point of test run it should be done by usual `Asserting`.